lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -2018,6 +2018,20 @@ module Resources
             {
               'valueString': 'test string',
               'url': 'custom-attribute-value'
+            },
+            {
```

Comment:
Wouldn't the CodeableConcept custom-attribute-value need to be in it's own custom-attribute extension with it's own custom-attribute-name? Right now it looks like it is combined with the custom-attribute for the string, which doesn't seem valid.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/812/commits/a48d17ea0f5da9977dd29a235ff36d3034cea772#diff-bec48e0a1467ab0b687a5ea45ebd8dcdf17aabfbf446f8e0aa5b0a6b7aa6ba4eR2021 and https://github.com/cerner/fhir.cerner.com/pull/812/commits/a48d17ea0f5da9977dd29a235ff36d3034cea772#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR573

Reply:
Thanks for making the change. It seems odd that both the valueString custom attribute and valueCodeableConcept would have the same custom-attribute-name. Can we use this example of a CodeableConcept custom attribute from c1941. 

"extension": [
                {
                    "id": "ENCNTR:4047481",
                    "valueString": "Conversation",
                    "url": "custom-attribute-name"
                },
                {
                    "valueCodeableConcept": {
                        "coding": [
                            {
                                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/100040",
                                "code": "4630114",
                                "display": "Register Patient",
                                "userSelected": true
                            }
                        ],
                        "text": "Register Patient"
                    },
                    "url": "custom-attribute-value"

Reply:
https://github.com/cerner/fhir.cerner.com/pull/812/commits/cbc395d45dc3606a7fd1028eb1003ee011afd134#diff-bec48e0a1467ab0b687a5ea45ebd8dcdf17aabfbf446f8e0aa5b0a6b7aa6ba4eL2027